### PR TITLE
GHC 8.4 libraries support

### PR DESCRIPTION
**Pull request by varosi**
If you're okay with those changes, please upload them to Hackage as well.

### CODE REVIEW

File: http-api-data.cabal [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name:            http-api-data
-version:         0.3.7.2
+version:         0.3.7.3
```
Comment (phadej) [inline http-api-data.cabal]: please don't touch versions. If the current version is ok with GHC-8.4.1 we'll make a revision and not an upload. (and for the future: if you touch `version:` then you have to touch `Changelog` too).

File: http-api-data.cabal [hunk 2]
```diff
@@ -28,8 +28,8 @@ tested-with:
 
 custom-setup
   setup-depends:
-    base >= 4.7 && <4.11,
-    Cabal >= 1.18 && <2.1,
+    base >= 4.7 && <4.12,
+    Cabal >= 1.18 && <2.2,
```
Comment (phadej) [inline http-api-data.cabal]: And as I pointed in other issue (iirc `cabal-doctest`) the next proper release of `Cabal` will be `2.2` not `2.1`.

Comment (varosi) [reply http-api-data.cabal]: Thanks, I'll fix this.

File: http-api-data.cabal (hunk 1 above)
Comment (varosi) [reply http-api-data.cabal]: Thanks! I'll fix this

**Comment by varosi** [comment]
phadej/cabal-doctest#35 is now merged into master.

**Comment by phadej** [comment]
See #71

**Comment by phadej** [comment]
Thanks @varosi for PR, this  time it was a little more involved than just bumping a bounds, yet thanks anyway!

**Pull request closed without merging**